app/models/hint.rb

```diff
@@ -1,5 +1,36 @@
+require 'stringex/core_ext'
```

Comment:
Might not: "I don't think it's required, but if you tried it without adding that and it didn't work I guess it is"